app/services/fixture_sweeper.rb

```diff
@@ -10,7 +10,7 @@ def sweep(stdout = $stdout, simulate: false)
 
     show_unused_fixtures(stdout)
 
-    return if simulate
+    exit(1) if simulate
```

Comment:
This should only exit with 1 if there is uncleaned fixtures?

Reply:
> This should only exit with 1 if there is uncleaned fixtures?

Yes, if you look at line 9, there is an early return if there are no unused fixtures:
```ruby
return stdout.puts 'No unused fixtures were found' if unused_fixtures.empty?
```